special-pages/tests/page-objects/special-error.js

```diff
@@ -254,6 +259,24 @@ export class SpecialErrorPage {
         ).toBeVisible();
     }
 
+    async showsMalwarePage() {
+        const { page } = this;
+        await expect(page.getByText('Warning: This site may put your personal information at risk', { exact: true })).toBeVisible();
+        await expect(
+            page.getByText(
+                'DuckDuckGo blocked this page because it may be distributing malware designed to compromise your device or steal your personal information. Learn more',
+                { exact: true },
+            ),
+        ).toBeVisible();
+        await this.showsAdvancedInfo();
+        await expect(
+            page.getByText(
+                'If you believe this website is safe, you can report an error. You can still visit the website at your own risk.',
+                { exact: true },
+            ),
+        ).toBeVisible();
+    }
+
```

Comment:
for future PRs, let's look into ariaSnapshots in playwright, i think it'll make these (excellent) tests easier to write